terraform/cloudfoundry/variables.tf

```diff
@@ -25,19 +25,18 @@ variable "router_cidrs" {
   }
 }
 
-variable "cell_cidr_all" {
-  description = "CIDR for all cell subnets"
-  default     = "10.0.32.0/20"
-}
-
-variable "cf_cidr_all" {
-  description = "CIDR for all cell subnets"
-  default     = "10.0.16.0/20"
+variable "aws_backing_service_cidrs" {
+  description = "CIDR for AWS backing service subnets indexed by AZ"
+  default     = {
+    zone0 = "10.0.52.0/24"
+    zone1 = "10.0.53.0/24"
+    zone2 = "10.0.54.0/24"
+  }
 }
 
-variable "router_cidr_all" {
-  description = "CIDR for all router subnets"
-  default     = "10.0.48.0/20"
+variable "aws_backing_service_cidr_all" {
+  description = "CIDR for all aws_backing_service subnets"
+  default     = "10.0.52.0/20"
```

Comment:
Having 20 mask bits seems to create a subnet much larger than needed. Should this be changed to 22 or 23?